tool/test-bin/pub_command_runner.dart

```diff
@@ -24,7 +24,8 @@ class ThrowingCommand extends PubCommand {
   @override
   String get description => 'Throws an exception';
 
-  bool get hide => true;
+  @override
```

Comment:
Nice catch - but let's actually not hide this.

This is a command in the test-executable, no need to hide